CareKitStore/CareKitStoreTests/Structs/TestOutcomeValue.swift

```diff
@@ -178,12 +178,15 @@ class TestOutcomeValue: XCTestCase {
 
     func testEqualityOfEncodings(outcome1: OCKOutcomeValue, outcome2: OCKOutcomeValue) {
         let encoder = JSONEncoder()
+        encoder.outputFormatting = .sortedKeys
         do {
             let encoded1 = try encoder.encode(outcome1)
             let encoded2 = try encoder.encode(outcome2)
-            XCTAssertEqual(encoded1, encoded2, "OCKOutcomeValue encoding inequality")
+            let decoded1 = try XCTUnwrap(String(data: encoded1, encoding: .utf8))
+            let decoded2 = try XCTUnwrap(String(data: encoded2, encoding: .utf8))
+            XCTAssertEqual(decoded1, decoded2, "OCKOutcomeValue decoding inequality")
```

Comment:
Modified this because https://github.com/carekit-apple/CareKit/actions/runs/8106082423/job/22155443798#step:5:457 failed sometimes in the Xcode 15.2 (didn't show in Xcode 14) related to encoding with `CareKitStoreTests.TestOutcomeValue
  testCodingForIdenticalEntries, XCTAssertEqual failed: ("66 bytes") is not equal to ("66 bytes") - OCKOutcomeValue encoding inequality`. The test has been updated to compare json strings just incase one of the properties are actually different, the test will show exactly what key/value isn't equal